roles/lnls-ans-role-sirius-hla/defaults/main.yml

```diff
@@ -0,0 +1,68 @@
+---
+# package state; use 'present' to make sure it's installed, or 'latest' if
+# you want to upgrade or switch versions using a new repo.
+sirius_apps_packages_state: present
```

Comment:
I intentionally reused sirius_apps variable names in sirius_hla. deploy tests fine in currents hosts. 
I guess in some situations this variable name reuse may be a problem. I separated variables for these two roles.

Reply:
Reusing variables will override the previous value, if any. We wouldn't be able to set different values for the roles if they were the same. They also override the `defaults` for all roles that use that variable and this would be misleading.

Other cases could include the variables inside `vars` directory being overriding by another one by the same name and effectively removing the default `vars` packages from installation of that role.